testem.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   launch_in_dev: [
     'Chrome'
   ],
-  browser_start_timeout: process.env.CI ? 60 : undefined,
+  browser_start_timeout: process.env.CI ? 120 : undefined,
```

Comment:
I kept the existing code but updated the value to match the blueprint.

Reply:
I think we could probably just set it to 120 all the time, like the new blueprint does.